### PR TITLE
[FIX] website_{}: show correct placeholder in search bar

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, in the mobile view, in the search bar of the website users(forum) and events leader board, the placeholder is shown as Search courses

Screenshot from 2023-03-14 08-45-55

after this commit, the wrong placeholder will be updated to Search users in the placeholder as in the normal view.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by niyasraphy** [comment]
hi @tde-banana-odoo ,
if you have time, can you review this.
thanks

**Comment by niyasraphy** [comment]
hi @tde-banana-odoo @flch-odoo ,
if you have time, can you review this.
thanks

**Comment by niyasraphy** [comment]
hi @tde-banana-odoo ,
if you have time, can you review this.
thanks in advance

File: addons/website_event_track_quiz/i18n/website_event_track_quiz.pot [hunk 1]
```diff
@@ -411,7 +411,7 @@ msgstr ""
 
 #. module: website_event_track_quiz
 #: model_terms:ir.ui.view,arch_db:website_event_track_quiz.leaderboard_search_bar
-msgid "Search courses"
+msgid "Search users"
```
Comment (tde-banana-odoo) [inline addons/website_event_track_quiz/i18n/website_event_track_quiz.pot:414]: Don't remove the old translation but add a new one. That way the old one is always available for people that don't upgrade their template, and the new one will be used for people that upgrade or create a new db in 14.

File: addons/website_profile/views/website_profile.xml [hunk 2]
```diff
@@ -59,7 +59,7 @@
                                 <div class="dropdown-menu dropdown-menu-right w-100" style="right: 10px;">
                                     <form class="px-3" t-attf-action="/profile/users" role="search" method="get">
                                         <div class="input-group">
-                                            <input type="text" class="form-control" name="search" placeholder="Search courses"/>
+                                            <input type="text" class="form-control" name="search" placeholder="Search users"/>
```
Comment (tde-banana-odoo) [inline addons/website_profile/views/website_profile.xml:62]: No translation to update in 'website_profile' module ?

**Review by tde-banana-odoo — COMMENTED** [review]
Seems legit indeed, just some technical remarks on translations :)

File: addons/website_event_track_quiz/i18n/website_event_track_quiz.pot (hunk 1 above)
Comment (niyasraphy) [reply addons/website_event_track_quiz/i18n/website_event_track_quiz.pot:419]: hi @tde-banana-odoo ,
updated the translation

File: addons/website_profile/views/website_profile.xml (hunk 2 above)
Comment (niyasraphy) [reply addons/website_profile/views/website_profile.xml:62]: hi @tde-banana-odoo ,
for the website_profile, the term is already existing in the pot file

**Comment by tde-banana-odoo** [comment]
@robodoo r+

Thanks for update !

**Comment by tde-banana-odoo** [comment]
@robodoo retry

**Pull request closed without merging**